NavigationLifecycleMonitor Parameter specified as non-null is null

Found a regression from https://github.com/mapbox/mapbox-navigation-android/pull/2185 conversion 👇 during testing.

```
10-14 11:44:57.097 28472-28472/com.mapbox.services.android.navigation.app E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.mapbox.services.android.navigation.app, PID: 28472
    java.lang.RuntimeException: Unable to start activity ComponentInfo{com.mapbox.services.android.navigation.app/com.mapbox.services.android.navigation.testapp.NavigationSettingsActivity}: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter bundle
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2416)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476)
        at android.app.ActivityThread.-wrap11(ActivityThread.java)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344)
        at android.os.Handler.dispatchMessage(Handler.java:102)
        at android.os.Looper.loop(Looper.java:148)
        at android.app.ActivityThread.main(ActivityThread.java:5417)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
     Caused by: java.lang.IllegalArgumentException: Parameter specified as non-null is null: method kotlin.jvm.internal.Intrinsics.checkParameterIsNotNull, parameter bundle
        at com.mapbox.services.android.navigation.v5.internal.navigation.NavigationLifecycleMonitor.onActivityCreated(NavigationLifecycleMonitor.kt)
        at android.app.Application.dispatchActivityCreated(Application.java:195)
        at android.app.Activity.onCreate(Activity.java:922)
        at android.preference.PreferenceActivity.onCreate(PreferenceActivity.java:536)
        at com.mapbox.services.android.navigation.testapp.NavigationSettingsActivity.onCreate(NavigationSettingsActivity.java:28)
        at android.app.Activity.performCreate(Activity.java:6251)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2476) 
        at android.app.ActivityThread.-wrap11(ActivityThread.java) 
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1344) 
        at android.os.Handler.dispatchMessage(Handler.java:102) 
        at android.os.Looper.loop(Looper.java:148) 
        at android.app.ActivityThread.main(ActivityThread.java:5417) 
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

Overridden Android parameters in `NavigationLifecycleMonitor` methods should be nullable.

cc @Lebedinsky

--- a/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/NavigationLifecycleMonitor.kt
+++ b/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/internal/navigation/NavigationLifecycleMonitor.kt
@@ -26,47 +26,51 @@ internal class NavigationLifecycleMonitor(
         initCurrentOrientation(application)
     }
 
-    override fun onActivityStarted(activity: Activity) {
-        val newOrientation = activity.resources.configuration.orientation
-        // If a new orientation is found, set it to the current
-        if (currentOrientation != newOrientation) {
-            currentOrientation = newOrientation
-            val currentTimeMillis = System.currentTimeMillis()
-            // If the current orientation is now landscape, add the time the phone was just in portrait
-            when (currentOrientation) {
-                Configuration.ORIENTATION_LANDSCAPE -> {
-                    portraitTimeInMillis += currentTimeMillis - portraitStartTime
-                }
-                Configuration.ORIENTATION_PORTRAIT -> {
-                    portraitStartTime = currentTimeMillis
+    override fun onActivityStarted(activity: Activity?) {
+        activity?.let {
+            val newOrientation = it.resources.configuration.orientation
+            // If a new orientation is found, set it to the current
+            if (currentOrientation != newOrientation) {
+                currentOrientation = newOrientation
+                val currentTimeMillis = System.currentTimeMillis()
+                // If the current orientation is now landscape, add the time the phone was just in portrait
+                when (currentOrientation) {
+                    Configuration.ORIENTATION_LANDSCAPE -> {
+                        portraitTimeInMillis += currentTimeMillis - portraitStartTime
+                    }
+                    Configuration.ORIENTATION_PORTRAIT -> {
+                        portraitStartTime = currentTimeMillis
+                    }
                 }
             }
         }
     }
 
-    override fun onActivityResumed(activity: Activity) {
+    override fun onActivityResumed(activity: Activity?) {
         resumes.add(System.currentTimeMillis())
     }
 
-    override fun onActivityPaused(activity: Activity) {
+    override fun onActivityPaused(activity: Activity?) {
         pauses.add(System.currentTimeMillis())
     }
 
-    override fun onActivityDestroyed(activity: Activity) {
-        if (activity.isFinishing) {
-            activity.application.unregisterActivityLifecycleCallbacks(this)
+    override fun onActivityDestroyed(activity: Activity?) {
+        activity?.let {
+            if (it.isFinishing) {
+                it.application.unregisterActivityLifecycleCallbacks(this)
+            }
         }
     }
 
     //region Unused Lifecycle Methods
 
-    override fun onActivityCreated(activity: Activity, bundle: Bundle) {
+    override fun onActivityCreated(activity: Activity?, bundle: Bundle?) {
     }
 
-    override fun onActivityStopped(activity: Activity) {
+    override fun onActivityStopped(activity: Activity?) {
     }
 
-    override fun onActivitySaveInstanceState(activity: Activity, bundle: Bundle) {
+    override fun onActivitySaveInstanceState(activity: Activity?, bundle: Bundle?) {
     }
 
     //endregion

